Jenkinsfile

```diff
@@ -887,11 +888,15 @@ pipeline {
                                                    mv $DAOS_BASE/install/bin/daos_admin \
                                                       $DAOS_BASE/install/bin/orig_daos_admin
 
+                                               # install junit-capable go test runner
+                                               curl -sSL "https://github.com/gotestyourself/gotestsum/releases/download/v${GTS_RELEASE}/gotestsum_${GTS_RELEASE}_linux_amd64.tar.gz" | \
```

Comment:
Should add `-f` to your `curl` command options.

Reply:
Well, if you really want to build from scratch, the project does publish source tarballs, e.g. [v0.4.0](https://github.com/gotestyourself/gotestsum/archive/v0.4.0.tar.gz).

I would imagine that there are rpm macros for go builds now, so it shouldn't be too hard to make an RPM out of this.